napari/utils/action_manager.py

```diff
@@ -288,18 +294,22 @@ def unbind_shortcut(self, name: str) -> Union[Set[str], None]:
 
     def _emit_shortcut_change(self, name: str, shortcut=''):
         tt = self._build_tooltip(name) if name in self._actions else ''
-        self.events.shorcut_changed(name=name, shortcut=shortcut, tooltip=tt)
+        self.events.shortcut_changed(name=name, shortcut=shortcut, tooltip=tt)
 
     def _build_tooltip(self, name: str) -> str:
         """Build tooltip for action `name`."""
         ttip = self._actions[name].description
+        is_toggle_on_long_hold = self._actions[name].is_toggle_on_long_hold
 
         if name in self._shortcuts:
             jstr = ' ' + trans._p('<keysequence> or <keysequence>', 'or') + ' '
             shorts = jstr.join(f"{Shortcut(s)}" for s in self._shortcuts[name])
-            ttip += f'({shorts})'
+            ttip += f' ({shorts})'
 
         ttip += f'[{name}]' if self._tooltip_include_action_name else ''
+        if is_toggle_on_long_hold:
+            ttip += trans._("\nHold shortcut to temporarily activate")
```

Comment:
With #5018 it will be possible to set an additional shortcut, so maybe reflect that here?
```suggestion
            ttip += trans._("\nHold shortcut(s) to temporarily activate")
```

Reply:
But users need only to hold one of the shortcuts; holding multiple may produce strange behavior.

Reply:
True
😂 
Edited: "hold a shortcut"
Could make it: "hold either shortcut" to make it abundantly clear 🤔

Reply:
I made a compromise here:

If there is only one shortcut, it will say `Hold shortcut to temporarily activate`
If there are multiple shortcuts it will say `Hold either shortcut to temporarily activate`

Reply:
To be honest, I don't think there's any risk of people purposefully pressing the two alternative shortcuts for the same action at the same time :P I think keeping consistent with the wording makes it easier to refactor/generalize later.